r4e-image.sh

```diff
@@ -73,31 +115,32 @@ EOF
 fi
 
 # create blueprint
-echo "creating blueprint $BLUEPRINT_FILE"
+BLUEPRINT_TEMPLATE=edgedevice-blueprint.tmpl
+BLUEPRINT_FILE=blueprint.toml
+echo "Creating blueprint $BLUEPRINT_FILE"
 envsubst < $BLUEPRINT_TEMPLATE > $BLUEPRINT_FILE
 composer-cli blueprints push $BLUEPRINT_FILE
 
 # create image
-echo "creating image $IMAGE_NAME"
-composer-cli compose start $IMAGE_NAME rhel-edge-commit > temp.out
-BUILD_ID=$(awk '{print $2}' temp.out)
+echo "Creating image $IMAGE_NAME"
+BUILD_ID=$(composer-cli -j compose start $IMAGE_NAME edge-commit | jq '.build_id')
 echo "waiting for build $BUILD_ID to be ready..."
-while [ "$(composer-cli compose status | grep $BUILD_ID | grep -c RUNNING)" != "0" ] ; do sleep 5 ; done
-if [ "$(composer-cli compose status | grep $BUILD_ID | grep -c FINISHED)" == "0" ] ; then
+while [ $(composer-cli -j compose status  | jq -r '.[] | select( .id == '$BUILD_ID' ).status') == "RUNNING" ] ; do sleep 5 ; done
```

Comment:
I had to add `| tr -d '"'` to the BUILD_ID in order to remove the quotas (' " ')

Reply:
same as above.